filter_plugins/trellis_filters.py

```diff
@@ -0,0 +1,20 @@
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import re
+
+def reverse_www(host=None):
+    ''' Add or remove the www subdomain '''
+    match = re.match(r'www\.(.+)', host)
+    if match:
+        return match.group(1)
+    return 'www.' + host
```

Comment:
Excellent, thank you. I see this in [PEP 08](https://www.python.org/dev/peps/pep-0008/#programming-recommendations):
```python
# No
def bar(x):
    if x < 0:
        return
    return math.sqrt(x)

# Yes
def foo(x):
    if x >= 0:
        return math.sqrt(x)
    else:
        return None
```

Another improvement is that although [re.match()](https://docs.python.org/2/library/re.html#re.match) will...
> Return `None` if the string does not match the pattern

I should probably change the evaluation to `if match is not None:` because [PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) also says this:
> beware of writing if `x` when you really mean `if x is not None` -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!